README.md

```diff
@@ -53,26 +53,30 @@ any new files in the repository.
 Make your own customizations
 ----------------------------
 
-Put your customizations in dotfiles appended with `.local`:
+Create a directory for your personal customizations: 
 
-* `~/.aliases.local`
-* `~/.git_template.local/*`
-* `~/.gitconfig.local`
-* `~/.gvimrc.local`
-* `~/.psqlrc.local` (we supply a blank `.psqlrc.local` to prevent `psql` from
+    mkdir ~/dotfiles-local
```

Comment:
Nice! Maybe mention the `rcrc` configuration option right before or after the first time `~/dotfiles-local` is referenced in this README? Then keep the rest as-is?